h2o-py/h2o/frame.py

```diff
@@ -2577,6 +2577,128 @@ def pivot(self, index, column, value):
             raise H2OValueError("'index' argument is not type enum, time or int")
         return H2OFrame._expr(expr=ExprNode("pivot",self,index,column,value))
 
+    def rank_within_group_by(self, group_by_cols, sort_cols, ascending=[], new_col_name="New_Rank_column", sort_cols_sorted=False):
+        """
+        This function will add a new column rank where the ranking is produced as follows:
+         1. sorts the H2OFrame by columns sorted in by columns specified in group_by_cols and sort_cols in the directions
+           specified by the ascending for the sort_cols.  The sort directions for the group_by_cols are ascending only.
+         2. A new rank column is added to the frame which will contain a rank assignment performed next.  The user can
+           choose to assign a name to this new column.  The default name is New_Rank_column.
+         3. For each groupby groups, a rank is assigned to the row starting from 1, 2, ... to the end of that group.
+         4. If sort_cols_sorted is TRUE, a final sort on the frame will be performed frame according to the sort_cols and
+            the sort directions in ascending.  If sort_cols_sorted is FALSE (by default), the frame from step 3 will be
+            returned as is with no extra sort.  This may provide a small speedup if desired.
+
+        :param group_by_cols: The columns to group on (either a single column name/index, or a list of column names
+          or column indices
+        :param sort_cols: The columns to sort on (either a single column name/index, or a list of column names or
+          column indices
+        :param ascending: Optional Boolean array to denote sorting direction for each sorting column.  True for
+          ascending, False for descending.  Default is ascending sort.  Sort direction for enums will be ignored.
+        :param new_col_name: Optional String to denote the new column names.  Default to New_Rank_column.
+        :param sort_cols_sorted: Optional Boolean to denote if the returned frame should be sorted accoring to sort_cols
```

Comment:
typo: "accoring" should be "according"

Reply:
Fixed the typo and the assignment in python from <- to =.  Thanks, Erin.